### PR TITLE
Generic kafka avro

### PR DESCRIPTION
**Pull request by jaffee**
genericizing ingest - running `pdk ingest` should allow you to ingest fairly arbitrary json or avro data.

### CODE REVIEW

**Comment by yuce** [comment]
It would be great to find replacements for `Parrrser` and `Mapppper` before it's too late :)

**Comment by jaffee** [comment]
they're going to become Parser and Mapper - I'm working on rewriting our old use cases in this new structure

File: cmd/ingest.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+			if err != nil {
+				return err
+			}
+			log.Println("Done: ", time.Since(start))
+			select {}
+		},
+	}
+	flags := ingestCommand.Flags()
+	flags.StringSliceVarP(&IngestMain.KafkaHosts, "kafka-hosts", "k", IngestMain.KafkaHosts, "Directory containing ingest data files.")
+	flags.StringSliceVarP(&IngestMain.KafkaTopics, "kafka-topics", "t", IngestMain.KafkaTopics, "Directory containing ingest data files.")
```
Comment (yuce) [inline cmd/ingest.go]: Missing description for hosts and topics.

**Comment by jaffee** [comment]
@yuce I ended up renaming Parrrrser and Mapppper to RecordParser and RecordMapper since they are parsers and mappers for a whole record rather than just one field. I'm closing this PR because it's superseded by the generic-improvements PR

**Pull request closed without merging**